### DISCUSSION TITLE
Key sequences?

### DISCUSSION THREAD

**Discussion by test3211234** [charmbracelet/bubbletea#868] [Q&A]
I was looking at [this key test file](https://github.com/charmbracelet/bubbletea/blob/a6f07b8ba6439fa65612a350bc1878d9d8c0447a/key_test.go#L265C2-L265C2) and saw both "a alt+a a" (separated by space) and "abcd" (no space separation) for what I assume is key sequences, as in press a then b then c then d in succession. My question is which is right and why the ambiguity? Shouldn't it be "a b c d"? Thanks

**Comment by gunererd** [comment]
@test3211234 did you find anything useful? I'm trying to implement a very simple modal editor inspired from vim and I want to make use of commands like dd, yy etc.

**Reply by test3211234** [reply]
Nope, was just a small question. I would say make it VSCode like and separate with pluses and separate strings with spaces

**Comment by bashbunni** [comment]
@test3211234 hey, you can do either way. No matter the format, they are all registered as key presses in bubble tea. Use whichever format you prefer
